spec/character.md

```diff
@@ -52,6 +52,18 @@ Keyword | Use
 `,`  | Separates arguments in collections or function or method signatures
 `_`  | Anonymous value
 
+## Reassignment Operations
+
+Keyword | Use
+---|---
+`+=`  | Add left side to right side and assign to left side
```

Comment:
Lots of left and right here.
Consider using `variable` to refer to the left side, and `value` to the right:
`+=` then has the description `Add value to variable, and assign to variable`
Perhaps the worst offender here becomes more readable:
`*=` has description `Raise variable to value, and assign to variable`